SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -633,6 +653,45 @@ MeasureCRTMVersion (
            );
 }
 
+// MU_CHANGE_217204
+
+/**
+Measure Source Debug Enabled.
+
+@retval EFI_SUCCESS           Operation completed successfully.
+@retval EFI_OUT_OF_RESOURCES  No enough memory to log the new event.
+@retval EFI_DEVICE_ERROR      The command was unsuccessful.
+
+**/
+EFI_STATUS
+MeasureSourceDebugEnabled (

```

Comment:
I would follow up with @cfernald to see if this can be adapted for mu_feature_debugger or should be dropped now as Mu is not using SourceLevelDebugPkg.

Reply:
I dont think we want to drop this, as we are still using the source debug enabled device state flag, right>

Reply:
@cfernald this does not tie into the device state flag (I think that would be a valid change for this). This tied into a SourceLevelDebugPkg library that just returned true or false. A platform would theoretically override it with the correct way to determine. With dropping SourceLevelDebugPkg, we don't have that lib any longer.

Reply:
As long as there are no potential callers in the codebase, I think we should drop it. The normal debug measurement to PCR7 will be made in Tcg2Dxe.

Reply:
Dropping